test/reduce/memory_table.wast

```diff
@@ -2,12 +2,15 @@
  (type $i (func (result i32)))
  (memory $0 256 256)
  (table 481 481 funcref)
+ (table 354 354 i31ref)
  (elem (i32.const 0) $f0 $f0 $f1 $f2 $f0 $f3 $f0)
+ (elem (table 1) (i32.const 0) i31ref (item (ref.i31 (i32.const 0))) (item (ref.i31 (i32.const 1))))
```

Comment:
If you make one of these values different than 0 and 1 then I think it will not be removed, which would show that we keep things that are needed.
```suggestion
 (elem (table 1) (i32.const 0) i31ref (item (ref.i31 (i32.const 0))) (item (ref.i31 (i32.const 42))))
```

Reply:
Done, thanks!